src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html

```diff
@@ -0,0 +1,213 @@
+      </li>
+    </ul>
+    <p>
+      After a session closes, you may:
+    </p>
+    <ul>
+      <li>
+        Publish results: make a session's results visible to students. Click the <button class="btn btn-sm btn-secondary dropdown-toggle">Results</button> dropdown on the <b>Sessions</b> page, then select <b>Publish Results</b>. Students will not be able to view the session's results until you publish them.
+      </li>
+      <li>
+        Download results of a session in spreadsheet format: first view the results of a session, then click <button class="btn btn-sm btn-primary">Download Results</button> to download the results of a session as a CSV file.
+      </li>
+    </ul>
+  </div>
+  <p align="right">
+    <button type="button" class="btn btn-info" (click)="jumpTo('top')">Back to Top <i class="fas fa-chevron-up"></i></button>
+  </p>
+  <div class="separate-content-holder">
+    <hr>
+  </div>
+  <h2 id="other-actions">5. Learn about other actions you can perform</h2>
+  <div>
+    <p>
+      Beyond allowing you to collect and disseminate feedback, TEAMMATES can be a useful repository of student and course information. You may use TEAMMATES to:
+    </p>
+    <ul>
+      <li>
+        <a href="/web/front/help/instructor#student-view-profile" target="_blank" rel="noopener noreferrer">View a student's profile</a> and <a href="/web/front/help/instructor#student-view-responses" target="_blank" rel="noopener noreferrer">all past records of a student</a>: view the profile that any enrolled student has written for him/herself, and see in one place all submissions given/received by a student. Handy for examining how a student progressed through a course.
+      </li>
+      <li>
+        <a href="/web/front/help/instructor#student-edit-details" target="_blank" rel="noopener noreferrer">Edit a student's data</a>: change a student's registered name, section or team name, or email address. You can also note down comments on students, for example to inform other instructors of information about a student that they should take note of.
+      </li>
+      <li>
+        <a href="/web/front/help/instructor#student-email" target="_blank" rel="noopener noreferrer">Email a group of students</a>: contact students regarding their feedback responses, or the course in general. Also handy for locating the email address of past students.
+      </li>
+      <li>
+        Search: <a href="/web/front/help/instructor#student-search" target="_blank" rel="noopener noreferrer">search for students, teams or sections</a>, or <a href="/web/front/help/instructor#session-search" target="_blank" rel="noopener noreferrer">search for questions, responses or comments</a>.
+      </li>
+      <li>
+        <a href="/web/front/help/instructor#course-archive" target="_blank" rel="noopener noreferrer">Archive old courses</a>: archive old courses that you no longer need actively.
```

Comment:
For all the links above (and wherever applicable):
- Change the `/web/front/help` to `../`
- Use `routerLink` instead of `href`? Angular router has support to specify fragment portion of the URL.